src/midiEditorCommands.cpp

```diff
@@ -502,6 +502,65 @@ void cmdMidiSelectNotes(Command* command) {
 	outputMessage(s);
 }
 
+void midiMoveToItem(int direction) {
+	HWND editor = MIDIEditor_GetActive();
+	MIDIEditor_OnCommand(editor, ((direction==1)?40798:40797)); // Contents: Activate next/previous MIDI media item on this track, clearing the editor first
+	MIDIEditor_OnCommand(editor, 40036); // View: Go to start of file
+	MIDIEditor_OnCommand(editor, NamedCommandLookup("_FNG_ME_SELECT_NOTES_NEAR_EDIT_CURSOR")); // SWS/FNG: Select notes nearest edit cursor
```

Comment:
While I realise the custom action relied on an SWS command and while it's generally recommended that SWS is installed, I try not to "break" things if the user doesn't have SWS installed. Can you confirm this doesn't crash or the like if SWS isn't installed? If this is a problem, working around this is probably as simple as checking the return from NamedCommandLookup and just not doing this part if it doesn't return an id. Unfortunately, the return value isn't explicitly documented for this case. I'm guessing 0, but it might also be -1.